PlayCover/Model/Keymapping.swift

```diff
@@ -9,6 +9,27 @@ import AppKit
 import Foundation
 import UniformTypeIdentifiers
 
+extension NSAlert {
+    static func differentBundleIdKeymap() -> Bool {
```

Comment:
To make it more consistent with other alerts, it should be moved to be a private function in the Keymapping class instead of having it as an extension of NSAlert.